src/main.cpp

```diff
@@ -72,7 +72,7 @@ void initialize() {
 
     // thread to for brain screen and position logging
     pros::Task screenTask([&]() {
-        lemlib::Pose pose(0, 0, 0);
+        lemlib::Pose pose(0_in, 0_m, 0_rad);
```

Comment:
```suggestion
        lemlib::Pose pose(0_in, 0_in, 0_rad);
```
Use inches because this is user-facing code